AdobeStockImage/etc/Model/AssetRepository.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model;
```

Comment:
I think this file has to be removed from the pull request

Reply:
Deleted